.github/workflows/build-demo.yaml

```diff
@@ -15,7 +15,7 @@ name: Build Demo App
 on:
   pull_request:
     paths:
-      - 'packages/components-angular/**'
+      - 'packages/intranet-header-workspace/**'
```

Comment:
Why add the suffix `-workspace`?

Reply:
Because this package IS actually an angular-workspace: https://angular.io/guide/file-structure